install.sh

```diff
@@ -10,9 +10,12 @@ sudo () {
 type curl >/dev/null 2>&1 || { echo >&2 "curl is required for installation of habitat, but was not found. Exiting."; exit 1; }
 
 curl https://raw.githubusercontent.com/habitat-sh/on-prem-builder/master/NOTICE
+echo
 
+curl https://raw.githubusercontent.com/habitat-sh/on-prem-builder/master/LICENSE-NOTICE
 echo
-read -r -p "Continue with installation? [y/N] " response
+
+read -r -p "Do you accept the terms of this license? Answering yes will proceed with the installation. [y/N] " response
```

Comment:
Is there some way to automate this response?

Reply:
Not currently, but that's a great idea.  I'll add that before this merges.